lib_nbgl/include/nbgl_use_case.h

```diff
@@ -223,7 +223,7 @@ void nbgl_useCaseReviewStreamingFinish(const char           *finishTitle,
 uint8_t nbgl_useCaseGetNbTagValuesInPage(uint8_t                          nbPairs,
                                          const nbgl_layoutTagValueList_t *tagValueList,
                                          uint8_t                          startIndex,
-                                         bool                            *tooLongToFit);
+                                         bool                            *flag);
```

Comment:
could be good to have a name more explicit for this param

Reply:
Do you have any recommendation for that name?

Reply:
`requireSpecificDisplay` or
`needSpecificDisplay` or 
`isSpecificDisplayRequired`

for example

Reply:
Done